README.md

```diff
@@ -4,27 +4,43 @@ Automating MakePlayingCards's online ordering system.
 The below guide describes the procedure for setting up the web component. If you're here to download the clientside program, check the Releases tab.
 
 # Requirements
-`requirements.txt` for the web application and local tool combined exists in the repo as well.
+`requirements.txt` for the web application and local tool combined.
 
-## Web application:
-JS libraries:
-* `bootstrap@4.6.0`
-* `jquery-ui-touch-punch@0.2.3`
-* `slick.js@1.8.1`
-
```

Comment:
There's probably an easy way to install the JS libraries that I missed. Might be helpful to explain what you do with these bits of text that actually installs these.

Reply:
I changed all the JS and CSS libraries to use CDNs rather than local copies, so while we can list these libraries in the README for info purposes, knowledge of them are not required for installation.